docs/bootstrap-application.md

```diff
@@ -0,0 +1,109 @@
+# Sparkle Web App
+
+Codebase for the Sparkle platform, brought to you by [Sparkle](https://sparklespaces.com/), a platform for the most immersive & interactive virtual events.
+
+### Connect your application to Firebase
+
+<!-- section 1 -->
+
+## 'Bootstrap' the platform
+
+Make sure that you're located in ./scripts folder.
+
+First of all you'd need to boostrap an environment using the following command:
+
+* Run the bootstrap script `./bootstrap-new-environment.ts TODO-PROJECT-ID [PRIVATE-KEY-FILE].json`
+
+As a result you should be able to see the 'Bootstraping complete' message
+<!--<!-- ![image](./images/creating-new-environments/57-bootstrap-complete.png) --> -->
+
+Then we'll be able to add a user to your application:
+
+```
+⇒ ./create-users.ts [PRIVATE-KEY-FILE].json user@sparkle.space
+
+User created: email=user@sparkle.space, password=REDACTED
+
+```
+
+We seem to need to manually create the `roles` collection in firestore, with a single `admin` document, that contains the following fields:
+
+* `allowAll` (boolean): `false`
+* `users` (array of strings): either an empty array (`[]`), or adding the user uid's copied from Firebase Authentication for the users who need to be admins.
+
+**Note:** you also need to be added to the `owners` field on a venue to be able to edit it. The `roles` collection just controls whether you're able to access the admin panel at all.
+
+<!-- ![image](./images/creating-new-environments/53-roles-collection.png) -->
+
+Once complete you should be able to make that user an admin. To do so run this command:
+* `./update-admin-role-users.ts [PRIVATE-KEY-FILE].json ADD user@sparkle.space`
+
+As a result you'll see this message:
+<!-- ![image](./images/creating-new-environments/58-user-admin-role.png) -->
```

Comment:
When showing the messages from script output, lets stick to showing it in text within a code block rather than images

Reply:
done to all the images 025e814